hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -655,12 +656,14 @@
 <%
   if (master.getAssignmentManager().isTableEnabled(table.getName())) {
     try {
-      CompactionState compactionState = admin.getCompactionState(table.getName()).get();
+      CompactionState compactionState = admin.getCompactionState(table.getName()).get(1, TimeUnit.SECONDS);
```

Comment:
1 sec is too short? And another way is: get the compaction state from master's memory and avoid rpc call.

Reply:
Thanks for the suggestion, let me try.